linter.py

```diff
@@ -85,20 +69,8 @@ def cmd(self):
 
         command.append('--json')  # need this for simpler error handling
 
-        c = self.build_cmd(command)
```

Comment:
This is and was a mistake. `cmd()` returns command which gets passed to `build_cmd` by SL later.